client/gutenberg/extensions/editor-notes/index.js

```diff
@@ -14,23 +16,16 @@ const attributes = {
 	},
 };
 
-const edit = ( { attributes: { notes }, className, isSelected, setAttributes } ) => (
-	<div className={ isSelected ? 'is-selected' : '' }>
-		{ ! isSelected && (
```

Comment:
Any good reason to be getting rid of this functionality? It was nice to have the notebook icon while editing the block content.